pkg/mover/chart.go

```diff
@@ -15,10 +15,23 @@ import (
 	"helm.sh/helm/v3/pkg/chartutil"
 )
 
-const DefaultRetries = 3
+const defaultRetries = 3
 
 type Logger interface {
 	Printf(format string, i ...interface{})
+	Println(i ...interface{})
```

Comment:
Not a big deal but https://go-proverbs.github.io/ `The bigger the interface, the weaker the abstraction.`

By using a single method we allow consumers to have simpler loggers, or more existent loggers that already fix the interface.

Most of the time this will be the standard library logger or Cobra logger, so it is fine anyways.

Reply:
I understand, but that single method was technically not enough, we were using a format based interpolation when there was nothing to interpolate. And as you said, Cobra, which is the main and probably the only custom logger client supports both methods

Reply:
Do you think in this specific example that the abstraction might be too weak? Would you recommend coming back to the previous, single method approach?

Reply:
This was a nitpick. It is fine.